pkg/fleet/installer/setup/djm/emr.go

```diff
@@ -0,0 +1,182 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package djm contains data-jobs-monitoring installation logic
+package djm
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/fleet/installer/setup/common"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"time"
+)
+
+const (
+	emrInjectorVersion     = "0.26.0-1"
+	emrJavaVersion         = "1.42.2-1"
```

Comment:
nit: since this is the tracer version (and not the version of java) 

```suggestion
	emrJavaTracerVersion         = "1.42.2-1"
```